src/main/java/htsjdk/samtools/util/SortingCollection.java

```diff
@@ -344,7 +345,7 @@ public void cleanup() {
                                                        final Comparator<T> comparator,
                                                        final int maxRecordsInRAM) {
 
-        final Path tmpDir = IOUtil.getDefaultTmpDirPath();
+        final Path tmpDir = Paths.get(System.getProperty("java.io.tmpdir"));
```

Comment:
should we worry about the case where this is null?

Reply:
There's no need, this is the same property name that java uses internally when you call APIs like `Files.createTempFile()`. If it fails then java is not configured correctly.

It would be nice if there was a non-magic string way to do it, but Java doesn't have a constant for property names like this.